notesapplication/notes/api_v1/serializers.py

```diff
@@ -1,21 +1,57 @@
-from ast import Not
-from dataclasses import field
-from django.contrib.auth.password_validation import validate_password
 from rest_framework import serializers
 from rest_framework.validators import UniqueValidator
+from rest_framework.pagination import PageNumberPagination
+from django.core.paginator import Paginator
 from ..models import Note, NoteVersion, Comment
 from datetime import date
 
+NUMBER_TO_GET_LATEST_COMMENT = 1
+
+
+class CommentSerializer(serializers.ModelSerializer):
+    """
+    CommentSerializer : Serializer for comment class and all operations of it
```

Comment:
Docstring could be improved as far as readability is concerned. Something like: *Responsible for nested Comments CRUD* or any related actions instead of `all operations`